src/apps/search/Place.tsx

```diff
@@ -132,6 +133,7 @@ const Place = () => {
         <LocationMarkers
           animate
           boundingBoxOptions={bboxOptions}
+          fitBoundingBox={_.get(config.search, "zoom_to_place", true)}
```

Comment:
Being nit-picky, but single quotes around the string value here.

Reply:
Oops! 😅